lsp-mode.el

```diff
@@ -176,7 +176,8 @@ Optional arguments:
                            :stderr stderr
                            :get-root ,get-root
                            :ignore-regexps ,ignore-regexps
-                           :ignore-messages ,ignore-messages)))
+                           :ignore-messages ,ignore-messages
+                           :restart-command (function ,enable))))
```

Comment:
Not sure whether the field should be called `:restart-command`, it doesn't really restart the server, but can only start it.

Reply:
It is only used to restart it, it is actually a reference to the function being defined in this macro.

Reply:
Yes, I see that :) But for future readers of the code, it should describe what it does, not how it's used or implemented.
How about renaming it to `:enable-function`? It's not required to be a command, and `enable` expresses better what it does.